QA.md

```diff
@@ -205,8 +205,14 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
     1. Close and open the wallet again and start a new wallet one without doing backup. It must show a yellow warning saying a backup must be done.
     1. Do the backup (following procedures in the 'Initialization' tests). The backup message has to disappear.
 
+1. **Hardware wallet migration**
+    1. Reset the wallet one more time with `localStorage.clear()`.
+    1. Uninstall the wallet and install the last version.
```

Comment:
```suggestion
    1. Uninstall the wallet and install the previous version.
```
suggestion: The word `last` is correct here, but could cause confusion when compared with `latest`. Maybe `previous` would be clearer

Reply:
Done